### PR TITLE
LLM Serving patterns first wave

### PR DESCRIPTION
**Pull request by ra2085**

### CODE REVIEW

File: llm-logging/llm-extraxt-candidates-v1/llm-extract-candidates-v1.xml
Comment (drush1980) [inline llm-logging/llm-extraxt-candidates-v1/llm-extract-candidates-v1.xml:1]: Parent folder is spelled incorrectly i.e. `llm-extraxt-candidates-v1` -> `llm-extract-candidates-v1`

File: llm-token-limits/llm_token_limits.ipynb [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "S478piq-SGLc"
+      },
+      "source": [
+        "## Test tiered AI products\n",
+        "\n",
+        "Apigee allows you to create a tiered product strategy with different API access levels (e.g., Bronze, Bronze, Goals) to cater to diverse user needs and limits. During the [Setup](#setup) stage you deployed 2 AI Product tiers for testing purposes.\n",
```
Comment (drush1980) [inline llm-token-limits/llm_token_limits.ipynb:164]: Was `(e.g., Bronze, Bronze, Goals)` supposed to be something like `(e.g., Bronze, Silver, Gold)`?

File: llm-token-limits/llm_token_limits.ipynb [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+      "metadata": {
+        "id": "NmRI6i4uhnJ4"
+      },
+      "source": [
+        "# 2000 tokens every 5 minutes\n",
+        "\n",
+        "This scenario demonstrates a basic interaction with a language model. The code repeatedly asks a language model the same question, \"Why is the sky blue?\" but phrased in different ways. It's a simple example of how to interact with a language model. After running the scenario **only onnce** expect the following behavior:\n",
+        "\n",
+        "* If using the **Bronze Key**, fhe final token count (sum of tokens from prompts and response candates) shouldn't exceed the Bronze AI Product tokens limit of 2000 tokens every 5 minutoes.\n",
+        "* If using the **Silver Key**, fhe final token count (sum of tokens from prompts and response candates) shouldn't exceed the Silver AI Product tokens limit of 5000 tokens every 5 minutoes.\n"
```
Comment (drush1980) [inline llm-token-limits/llm_token_limits.ipynb:210]: `candates` -> `candidates`?
`minutoes` -> `minutes`

File: llm-token-limits/llm_token_limits.ipynb [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZjW8UBEnZg9U"
+      },
+      "source": [
+        "# 5000 tokens every 5 minutes\n",
+        "\n",
+        "This scenario demonstrates a basic interaction with a language model. The code repeatedly asks a language model the same question, \"Why is the sky blue?\" but phrased in different ways to make sure the candidate responses are very extensive (high token count). After running the scenario **only onnce** expect the following behavior:\n",
```
Comment (drush1980) [inline llm-token-limits/llm_token_limits.ipynb:242]: `onnce` -> `once`

File: llm-token-limits/llm_token_limits.ipynb [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+        "\n",
+        "* If using the **Bronze Key**, fhe final token count (sum of tokens from prompts and response candates) **should exceed** the Bronze AI Product tokens limit of 2000 tokens every 5 minutoes. SHoud should expect `HTTP 429` error messages in the notebook and also visible on Apigee's debug session.\n",
```
Comment (drush1980) [inline llm-token-limits/llm_token_limits.ipynb:244]: `candates` -> `candidates`?
`minutoes` -> `minutes`
`SHoud should` -> `Should`

File: llm-token-limits/llm_token_limits.ipynb [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+        "* If using the **Silver Key**, fhe final token count (sum of tokens from prompts and response candates) shouldn't exceed the Silver AI Product tokens limit of 5000 tokens every 5 minutoes."
```
Comment (drush1980) [inline llm-token-limits/llm_token_limits.ipynb:245]: `candates` -> `candidates`?
`minutoes` -> `minutes`

**Review by drush1980 — CHANGES_REQUESTED** [review]

**Review by ra2085 — COMMENTED** [review]
applied all proposed fixes.

**Review by drush1980 — APPROVED** [review]
lgtm

**Pull request merged**